src/components/calcite-button/usage/as-anchor.md

```diff
@@ -0,0 +1,6 @@
+```html
+<calcite-button href="#content">Internal anchor</calcite-button>
+<calcite-button href="https://www.esri.com" icon-start="launch" rel="noopener noreferrer" target="_blank">
```

Comment:
The more I look at this code, the more I think we should just have a slotted anchor to achieve this. Do we really want to add all the anchor props to the calcite button in order to support them or should we just allow a user to slot a link?

Reply:
The problem right now is that `rel="noopener noreferrer" target="_blank"` are not valid documented props for calcite-button

Reply:
Gotcha @driskull I'll remove this usage suggestion (just realized this use case is already covered by internals.md)